### DISCUSSION TITLE
The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_TheiaIpgMpVf/chrome-sandbox is owned by root and has mode 4755.

### DISCUSSION THREAD

**Discussion by razorree** [eclipse-theia/theia#13941] [General] [answered]
`The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_TheiaIpgMpVf/chrome-sandbox is owned by root and has mode 4755.`

Message when I try to run `TheiaIDE.appimage`
It looks like some permissions are not set correctly when releasing Electron app ?

**Comment by msujew** [comment, marked as answer]
Hey @razorree,

Please see the issue report over at https://github.com/eclipse-theia/theia-blueprint/issues/363. This repository is for the Theia framework, not the IDE.

**Comment by razorree** [comment]
Thank You, i think we can close/delete it
